test/drafts/Strings.test.js

```diff
@@ -0,0 +1,49 @@
+const { constants } = require('openzeppelin-test-helpers');
+
+const StringsMock = artifacts.require('StringsMock');
+
+contract('Strings', function () {
+  const a = 'AAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
+  const b = 'BBBBBBBBBBBBBBBBBBBBBBB';
+  const ab = a + b;
+  const aa = a + a;
+  const empty = '';
+  const zero = 0;
+  const positiveNumber = 3195;
+
+  beforeEach(async function () {
+    this.strings = await StringsMock.new();
+  });
+
+  describe('concatenate', function () {
+    it('concatenates strings', async function () {
+      (await this.strings.concatenate(a, b)).should.equal(ab);
+    });
+
+    it('concatenates a string with itself', async function () {
+      (await this.strings.concatenate(a, a)).should.equal(aa);
+    });
+
+    it('concatenates with the empty string', async function () {
+      (await this.strings.concatenate(a, empty)).should.equal(a);
+    });
+
+    it('concatenates two empty strings', async function () {
+      (await this.strings.concatenate(empty, empty)).should.equal(empty);
+    });
+  });
+
+  describe('from uint256', function () {
+    it('converts 0', async function () {
+      (await this.strings.uint256ToString(zero)).should.equal(zero.toString());
+    });
+
+    it('converts a positive number', async function () {
+      (await this.strings.uint256ToString(positiveNumber)).should.equal(positiveNumber.toString());
```

Comment:
```suggestion
      (await this.strings.uint256ToString(4132)).should.equal('4132');
```

I think simple tests like this one benefit from not trying to be too smart and using global variables, etc. I'd apply this same suggestion to all other tests (e.g. replace `empty` for an `''`).

Reply:
Great have updated the tests.